docs/pages/blog/mui-x-v6.md

```diff
@@ -0,0 +1,188 @@
+---
+title: MUI X v6 is ready for production!
```

Comment:
I think that it would make more sense with. "ready for production!" makes sense from the context of the team and people using the beta, but it's likely a niche audience

```suggestion
title: Introducing MUI X v6.0.0
```